codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -166,6 +166,13 @@ public void goAwayReceived(int lastKnownStream, long errorCode, ByteBuf debugDat
                 listener.onGoAwayReceived(lastKnownStream, errorCode, debugData);
             }
         }
+
```

Comment:
Let's get rid of the check for `if (!alreadyNotified)` above.  I think we want to notify any time we receive a GOAWAY.

Same for `goAwaySent`